lib/Dialect/Triton/IR/Utility.cpp

```diff
@@ -0,0 +1,20 @@
+#include "triton/Dialect/Triton/IR/Utility.h"
+#include "triton/Dialect/Triton/IR/Dialect.h"
+
+using namespace mlir;
+namespace tt = mlir::triton;
+
+// Combine the current mask with the given predicate.
```

Comment:
nit: move the doc to the header

Reply:
done, thanks for the review @ThomasRaoux!